### PR TITLE
Bump package versions due to vulnerability issues

### PR DESCRIPTION
**Pull request by opravil-jan**
Hello, 

I upgraded packages so the vulnerability issues caused by old versions went away.
Is it needed anything else to make this PR merged?

Thanks
Jan

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
     "name": "eslint-plugin-mocha",
-    "version": "10.0.3",
+    "version": "10.0.4",
```
Comment (lo1tuma) [inline package.json:3]: Please don’t increase the package version in this PR. Releases will be done from the `master` branch once all changes have been merged.

**Review by lo1tuma — CHANGES_REQUESTED** [review]
Thanks! Please remove the version bump of `eslint-plugin-mocha` itself from this PR (in `package.json` and `package-lock.json`. Then I’m happy to merge this and publish a new release.

File: package.json (hunk 1 above)
Comment (opravil-jan) [reply package.json]: Done

Comment (lo1tuma) [reply package.json]: Thank you.

**Review by lo1tuma — APPROVED** [review]

**Pull request merged**